### DISCUSSION TITLE
Is it safe to update OctoPrint version above 1.8.7?

### DISCUSSION THREAD

**Discussion by jtulak** [fredrikbaberg/3dprinter-addons#89] [Q&A] [answered]
Hi. The previous version of your addon mentioned that OctoPrint above 1.8.x doesn't work without workarounds. I just installed this plugin and I see that it comes with OctoPrint 1.8.7. But 1.9.3 is available. Is it (reasonably) safe to keep OctoPrint up to date, or is there some known problem with going above 1.8.7?

**Comment by fredrikbaberg** [comment, marked as answer]
Hi!
As far as I know it should be safe to upgrade. IIRC the issue was with Ingress, but that should not be an issue with this version of the add-on. That said, a snapshot/backup is always a good idea.

If you encounter any issue please let me know.

**Reply by jtulak** [reply]
Thank you. I run the update to 1.9.3 and on a glance, it looks ok. I'm definitely still able to access it.
